composer.json

```diff
@@ -1,41 +1,44 @@
 {
     "name": "opencfp/opencfp",
     "license": "MIT",
+    "config": {
+        "sort-packages": true
+    },
     "require": {
         "php": ">=5.5",
-        "twig/twig": "1.*",
-        "pagerfanta/pagerfanta": "1.0.*@dev",
+        "aptoma/twig-markdown": "0.2.*",
         "cartalyst/sentry": "2.1.*",
```

Comment:
I added the `config` section above, then I ran

```
$ composer require "cartalyst/sentry:2.1.*"
```

to trigger the sorting, as this is the only package that isn't outdated.

```
$ composer info --direct | grep cartalyst/sentry
cartalyst/sentry              v2.1.6             PHP 5.3+ Fully-featured Authentication & Authorization System

$ composer outdated | grep cartalyst/sentry

Package fabpot/php-cs-fixer is abandoned, you should avoid using it. Use friendsofphp/php-cs-fixer instead.
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.
```